packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -109,13 +109,12 @@ Okta offers a variety of functions to manipulate attributes or properties to gen
 | `String.toUpperCase`     | (String input)                                                | String      | `String.toUpperCase("This")`                                                                                  | THIS           |
 | `String.toLowerCase`     | (String input)                                                | String      | `String.toLowerCase("ThiS")`                                                                                  | this           |
 
-
-The following <ApiLifecycle access="deprecated" /> functions perform some of the same tasks as the ones in the above table.
+The following <ApiLifecycle access="deprecated" /> functions perform some of the same tasks as the ones in the previous table.
 
 | Function                          | Example                             | Input         | Output    |
 | --------                          | ---------                           | -------       | --------  |
 | `toUpperCase(string)`             | `toUpperCase(source.firstName)`     | Alexander     | ALEXANDER |
-| `toLowerCase(string)`             | `toLowerCase(source.firstName)`     | AlexANDER     | alexander |
+| `toLowerCase(string)`             | `toLowerCase(source.firstName)`     | AlexANDER     | oleander |
```

Comment:
Was this change intentional?

Reply:
No. Thanks for the catch. Inadvertent typo.